src/js/components/ResortCardLocation/ResortCardLocation.js

```diff
@@ -0,0 +1,49 @@
+import { CImage } from '@coreui/react';
+import React from 'react';
+import useResortFlagImage from '../../hooks/useResortFlagImage';
+import { locationType } from '../../types/types';
+
+const ResortCardLocation = ({ location }) => {
+  const {
+    city,
+    country: {
+      name: countryName,
+      code: countryCode,
+    },
+    state,
+  } = location;
+
+  const flag = useResortFlagImage(countryCode);
```

Comment:
```suggestion
  const flagImageUrl = useResortFlagImage(countryCode);
```

Pays to be declarative